config.md

```diff
@@ -25,6 +25,8 @@ For example, if an implementation is compliant with version 1.0.1 of the spec, i
 **`root`** (object, required) configures the container's root filesystem.
 
 * **`path`** (string, required) Specifies the path to the root filesystem for the container.
+  The path MUST be an a relative path (not starting with /), which is relative to the bundle. e.g.
```

Comment:
However, I think it does make sense to [clarify the base from which relative paths are interpreted][1], in cases where someone does use a relative path here.  I'd recommend doing that with a stand-alone section early in this file, so other relative-path locations can benefit from it.  That would allow us to relax some absolute-path requirements we're still carrying around (e.g. for [namespace paths][2]).

[1]: https://github.com/opencontainers/runtime-spec/issues/509#issuecomment-228414738
[2]: https://github.com/opencontainers/runtime-spec/blob/22c29c18eddc75ccee9f6b202f61524f29653c5f/config-linux.md#namespaces